dtreeviz/trees.py

```diff
@@ -131,7 +131,7 @@ def leaf_sizes(self,
 
 
     def ctree_leaf_distributions(self,
-                                 display_type: str = "plot",
+                                 display_type: ("plot", "text") = "plot",
```

Comment:
I'm not sure I'm a big fan of plotting versus getting a string back (or printing). Seems like you would want something that plots and another function that returns Data structure indicating leaf id -> count, right?

Reply:
yes, indeed. This was from beginning :)

the "text" version was my solution from that old time in case the plot contains too many leaves and hard to get the node id.

Reply:
oh "text" is already in there and we just updated the type? ok.

Reply:
yes, I just put there the possible options the user could choose.